sscanss/app/dialogs/preferences.py

```diff
@@ -249,6 +249,36 @@ def generalForm(self):
         frame.setLayout(main_layout)
         self.stack.addWidget(create_scroll_area(frame))
 
+    def renderingSizeSlider(self, key, value, range=(5, 100)):
```

Comment:
Please upgrade this function to a class similar to the `ColourPicker` with a good name maybe "SliderTextInput" then move to _sscanss/core/util/widgets.py_ and import here.